ucp/_libs/utils.pyx

```diff
@@ -43,44 +45,55 @@ def get_buffer_data(buffer, check_writable=False):
     return data_ptr
 
 
-def get_buffer_nbytes(buffer, check_min_size, cuda_support):
+cpdef size_t get_buffer_nbytes(buffer, check_min_size, bint cuda_support) except *:
```

Comment:
Not sure on the typing of `check_min_size` here:
```suggestion
cpdef size_t get_buffer_nbytes(object buffer, object check_min_size, bint cuda_support) except *:
```

Reply:
Cython already defaults to `object` if a type is specified. So we are ok here.

Yeah `check_min_size` could be several things. So `object` is probably as close as we can get.